docker/service/src/main/resources/application.yml

```diff
@@ -1,7 +1,16 @@
 server:
-    port: ${SCR_SERVER_PORT:9000}
-    tomcat:
-        relaxed-query-chars: "|"
+  port: ${SCR_SERVER_PORT:9000}
+  tomcat:
+    relaxed-query-chars: "|"
```

Comment:
Missing from sandbox cfg

Reply:
not valid anymore - using the same codebase for prod and sandbox